applications/mne_anonymize/mne_anonymize.pro

```diff
@@ -69,12 +69,14 @@ CONFIG(debug, debug|release) {
 DESTDIR = $${MNE_BINARY_DIR}
 
 SOURCES += main.cpp \
-    settingscontroller.cpp \
-    fiffanonymizer.cpp
+    fiffanonymizer.cpp \
+    mainwindow.cpp \
+    settingscontrollerCL.cpp
 
 HEADERS  += \
-    settingscontroller.h \
-    fiffanonymizer.h
+    fiffanonymizer.h \
+    mainwindow.h \
+    settingscontrollerCL.h
```

Comment:
`settingscontrollerCL.h` should be `settingscontrollercl.h` captial letters in file names are discouraged.

Reply:
done